src/m365/pp/base/PowerPlatformCommand.ts

```diff
@@ -0,0 +1,11 @@
+import Command from '../../../Command';
+
+export default abstract class PowerPlatformCommand extends Command {
+  protected get graphResource(): string {
```

Comment:
Typically, each base class defines only one resource. Will we need Graph for all PP commands or only for `managementapp add`? If the latter, then I'd suggest that we move `graphResource` away from here and keep the base class aligned with other base classes.

Reply:
Just checked and the graph seems to be present in 2 or 3 commands in PowerShell; so lets agree to remove for consistency.